roles/translator/src/job_negotiator/message_handler.rs

```diff
@@ -0,0 +1,29 @@
+use crate::job_negotiator::JobNegotiator;
+use roles_logic_sv2::{
+    handlers::{job_negotiation::ParseServerJobNegotiationMessages, SendTo_},
+    job_negotiation_sv2::SetCoinbase,
+    parsers::JobNegotiation,
+};
+use tracing::info;
+pub type SendTo = SendTo_<JobNegotiation<'static>, ()>;
+use core::convert::TryInto;
+use roles_logic_sv2::{bitcoin::TxOut, errors::Error};
+
+impl ParseServerJobNegotiationMessages for JobNegotiator {
+    fn handle_set_coinbase(&mut self, message: SetCoinbase) -> Result<SendTo, Error> {
+        info!(
+            "Received allocate mining job token success message: {:?}",
+            message
+        );
+        let txout = TxOut {
+            value: self.coinbase_reward_sat,
```

Comment:
I would love to understand why we just hard-code this.

Reply:
this will be changed in next PR where we revert the JN to the one that is in the spec (this is the modified proposed and refused version) where coinbase is only managed by pool and not by proxy.